### DISCUSSION TITLE
Home Assistant OS 11.2

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2964] [Announcements]
## Changes

* Prepare 11.2.rc1 (#2939) @agners
* Bump rpi-rf-mod dt overlay to latest version (#2921) @jens-maus
* Bump buildroot to v2023.02.7 (#2936) @sairon
* Bump docker/build-push-action from 5.0.0 to 5.1.0 (#2935) @dependabot
* Bump actions/github-script from 6 to 7 (#2934) @dependabot
* fix: disable uas for wavlink adaptor (#2924) @bertrandgressier
* Bump dessant/label-actions from 3.1.0 to 4.0.0 (#2922) @dependabot
* Bump buildroot to add RTL885x firmware (#2911) @sairon
* Fix incorrect uppercase Y kconfig use (#2908) @jens-maus
* Refresh all boards' defconfigs (#2895) @sairon
* generic-x86-64: Add missing dependency for #2800 to kernel.config (#2813) @xtsrthm
* Maintain and upload artifacts index (#2839) @agners

## Home Assistant Operating System

* Fix Supervisor image corruption detection (#2953) @agners
* Use kernel local version for HAOS compiled Linux kernel (#2912) @agners
* Improve Bluetooth cache cleanup command (#2906) @agners
* Bump buildroot to update Docker to v24.0.7 (#2883) @agners

## Build System

* Revert "Upload artifacts index even on partial builds (#2901)" (#2905) @agners
* Upload artifacts index even on partial builds (#2901) @agners
* Add pruning of object file cache (#2899) @sairon
* Fix Artifacts index bump when using workflow_call (#2900) @agners
* Pass secrets to artifacts index generation workflow (#2893) @agners
* Improve handling of timeouts in tests (#2890) @sairon
* Checkout source code when uploading artifacts index (#2892) @agners
* Fix the condition for OS artifact download (#2891) @sairon
* Fix quotes in artifact index generation (#2889) @agners
* Separate artifacts index update into separate workflow (#2888) @agners
* Add a flag to determine tests were called from another workflow (#2887) @sairon
* Bump HAOS beta when stable version is released (#2885) @sairon
* Always use environment for bump_version job (#2884) @sairon
* Add test suite for Supervisor tests (#2880) @sairon
* Run tests within build workflow (#2882) @sairon
* Use the correct secret to clear cache on the Cloudflare frontend (#2881) @agners
* Print object cache statistics before upload (#2874) @agners
* Install AWS CLI tools in final build step (#2873) @agners
* Optimize build cache for dev builds (#2872) @sairon
* Add checkout step to bump_version job in the build workflow (#2871) @sairon
* Fix GitHub action build.yaml (#2868) @agners

## Raspberry Pi

* Do not start hciuart.service/bthelper@hci0.service if krnbt is used (#2947) @agners
* Bump buildroot to fix rpi-firmware hash (#2930) @sairon
* Bump Raspberry Pi firmware to latest stable version (#2929) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231030 (#2904) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231024 (#2864) @sairon

## Home Assistant Yellow

* Remove kernel HCI driver if no WiFi/Bluetooth module present (#2944) (#2948) @agners
* Do not start hciuart.service/bthelper@hci0.service if krnbt is used (#2947) @agners
* Fix miniUART on Yellow with kernel HCI init enabled (#2931) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* RaspberryPi: Update kernel 6.1.58 - stable_20231030 (#2904) @sairon

## Home Assistant Green

* Linux: Update kernel 6.1.63 (#2937) @sairon

## Open Virtual Appliance

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Generic x86-64

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Hardkernel ODROID

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Reduce Tinker and Amlogic kernel configs to defconfigs (#2925) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## ASUS Tinker

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Reduce Tinker and Amlogic kernel configs to defconfigs (#2925) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Khadas VIM Series

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon

## Generic aarch64

* Linux: Update kernel 6.1.63 (#2937) @sairon
* Clean up inconsistencies in kernel configs (#2923) @sairon
* Generic AArch64: Enable options required for virtiofs (#2909) (#2910) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.2'>Home Assistant OS 11.2</a>.</em>

**Comment by andrews63** [comment]
nice updates, will you guys be looking at Raspberry Pi5?

**Reply by sairon** [reply]
Sure, see https://github.com/home-assistant/operating-system/pull/2914 and https://github.com/home-assistant/operating-system/discussions/2844

**Comment by codecs02-marcher** [comment]
did you guys fix the apple remote widget not showing in iOS 17 for Universal Media Player.
the issue is not with smartIR media player but only with Universal Media Player.

**Comment by AdamGit69** [comment]
Been running HA for a couple years now, I have a VERY large setup with over 100 devices including multiple displays around the house, so I consider myself pretty proficient with HA and it's workings, however this is the first update of either HA or HA OS that has failed and killed my system. My Pi won't boot at all as far as I can tell anymore, I can't ping it, can't ssh nothing... Hit update and HA shutdown after a while as per normal but this time, it just never came back... Zip, Nadda, Nothing... Not sure where I can go from here other than have to start with a fresh install then try restore from a backup.. Sigh...

**Reply by agners** [reply]
We've updated the firmware and kernel to what Raspberry Pi offered as the latest update, so that might be the culprit :cry: 

Obviously, we do test new releases on all Raspberry Pi variants, even with different setups (with USB SSD/SD card etc.). However, we can't test every configuration which is out there.

Best is to open a [new issue](https://github.com/home-assistant/operating-system/issues/new/choose) along with all information such as which version of the Raspberry Pi you are using, USB SSD if used and its type etc. *Update:* Also, can you connect a display to the HDMI port to see if something shows up?

**Reply by AdamGit69** [reply]
Well. thank you so much for your response and sorry if I was blunt or rude in my original post, I was a tad flustered when I posted as my whole house died (for the first real time in a couple years) the night before I had guests arriving and besides it being a royal pain as I run a lot of automations, I really wanted to show it all off! lol 

Anyway, turns out, it was me or atleast my end that was the cause! My fairly new SD card ( I replace them every 6 months or so and this one was only 2 months old) had some corruption and that caused the failed update from what I can tell.  I chucked in a new card and after a hour or so, I was back up and running pretty quick and painless! So glad I backup every 2 days and not just onto the SD card! 

P.S. HA rocks and so does the entire dev team. Thank you all for your amazing work! :)

**Comment by xelemorf** [comment]
HAOS 11.2 upgrade on VMware OVA was updating for more than 2 hours for me, the upgrade was initiated after a clean reboot, it was **eventually successful**. Usually these would take less than 15 minutes.
_(VM has plenty of available free resources from the host: 2 vCPU /from a 12 thread CPU/ and 8GB of RAM, NVMe SSD based vDisk)_

**Comment by cedricdelhaes** [comment]
Hello guys, just a question about haos_rpi2-11.2.img.xz. I was excited to test HAOS on my old raspberry but the OS seem to not work. I see that the raspberry 2 is not supported anymore but in this case why there is an image available ?
